presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -27,9 +27,16 @@ Linux Operating System
 Java Runtime Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-* 64-bit required
-* version 11.0.7+ (version 12+ not supported)
-* Azul Zulu recommended (most tested)
+Presto requires a 64-bit version of Java 11, with a minimum required version of 11.0.7.
+Newer patch versions such as 11.0.8 or 11.0.9 are supported. Earlier patch versions
+such as 11.0.2 will not work, nor will earlier major versions such as 8, 9, or 10.
```

Comment:
as 11.0.2. do not work, nor do earlier major versions such as Java 8. 

I think we can leave 9 and 10 out since they are not LTS anyway.

Reply:
Agreed, saying "Java 8" is probably better than just "8" since it jumps out